openverse_catalog/dags/common/dag_factory.py

```diff
@@ -158,7 +158,10 @@ def create_provider_api_workflow(
             python_callable=_push_output_paths_wrapper,
             op_kwargs=pull_kwargs,
             depends_on_past=False,
-            execution_timeout=dagrun_timeout if dated else None,
```

Comment:
It seems reasonable to me that we'd want the timeout across the board, but I'm curious why it was previously applied only for dated data pulls? 🤔

Reply:
I'm not sure! This is [how it was prior to my interactions with the codebase](https://github.com/WordPress/openverse-catalog/blob/7ee62451343ee04a895290b23df9d9c79703dc74/openverse_catalog/dags/util/dag_factory.py#L77-L81) and I wanted to preserve behavior when I refactored this.

Reply:
Mysterious! 😄 At any rate I don't see any harm in adding the timeout to the undated runs. Timeouts can always be adjusted.

Reply:
Exactly! I think while we're activating the DAGs, we'll probably need to go back and adjust some of the timeouts (for instance, I know that Freesound will likely take quite a while to process).

Reply:
This makes sense to me too FWIW 😄